connectd/connectd.c

```diff
@@ -1483,6 +1492,24 @@ static void add_gossip_addrs(struct wireaddr_internal **addrs,
 
 	/* Wrap each one in a wireaddr_internal and add to addrs. */
 	for (size_t i = 0; i < tal_count(normal_addrs); i++) {
+		/* add TOR addresses in a second loop */
```

Comment:
Weird that you put "Changelog-fixed: connectd: do not try address hint twice" instead of "Peter to connect to non-Tor addresses"? (Or both, but the former seems like a trivial change?)